src/neuroconv/datainterfaces/ophys/miniscope/miniscopeconverter.py

```diff
@@ -0,0 +1,54 @@
+from ... import MiniscopeBehaviorInterface, MiniscopeImagingInterface
+from ....nwbconverter import NWBConverter
+from ....utils import FolderPathType
+
+
+class MiniscopeConverter(NWBConverter):
+    """Primary conversion class for handling Miniscope data streams."""
+
+    data_interface_classes = dict(
+        Miniscope=MiniscopeImagingInterface,
+        BehavCam=MiniscopeBehaviorInterface,
+    )
+
+    @classmethod
+    def get_source_schema(cls):
+        source_schema = super().get_source_schema()
+        source_schema.update(additionalProperties=True)
```

Comment:
Remind me why it was necessary to allow additional properties here?

Reply:
We can discuss this today but basically I get this error when trying to init the converter using `"folder_path"`:
`jsonschema.exceptions.ValidationError: Additional properties are not allowed ('folder_path' was unexpected)`
I thought it's better to use `"folder_path"` compared to `"source_data"` since both the imaging and the behavior data streams use the same folder.

Reply:
Oh, well sure - but then shouldn't we define the schema for the `folder_path` as a required property then? Via the same mechanism the interfaces use (`get_schema_from_method_signature`)?

I point this out because `additionalProperties=True` is a blanket shortcut used to bypass more rigorous specification, and so should be used only when truly necessary

Reply:
Ah sure, didn't think of `get_schema_from_method_signature`. Thank you!